### PR TITLE
Story1 wireframe and setup

### PR DESCRIPTION
**Pull request by alexjames4**
This has our initial empty files along with a source folder containing a wireframe and an attribution free image of a caterpillar that we shall incorporate into our menu screen as part of story 2: game screen.

Thanks

### CODE REVIEW

File: src/.DS_Store
Comment (mporam) [inline src/.DS_Store:1]: remove this file

File: src/Screenshot 2023-08-14 at 15.45.55 (2).png
Comment (mporam) [inline src/Screenshot 2023-08-14 at 15.45.55 (2).png:1]: please rename this file, that name is meaningless

Comment (mporam) [reply src/Screenshot 2023-08-14 at 15.45.55 (2).png:1]: also why is it a screenshot?

**Review by mporam — CHANGES_REQUESTED** [review]

File: src/.DS_Store
Comment (alexjames4) [reply src/.DS_Store:1]: I have removed this now

File: src/Screenshot 2023-08-14 at 15.45.55 (2).png
Comment (alexjames4) [reply src/Screenshot 2023-08-14 at 15.45.55 (2).png:1]: agreed, I have now changed it to two files. one for the mobile one for the desktop based off taken from Colin's Figma page.

They are still screenshots, but trimmed as the way figma was exporting the files weirdly as pdf

**Review by mporam — APPROVED** [review]

**Pull request merged**